### DISCUSSION TITLE
Isn't this supposed to be running every 5 minutes? Are we doing something wrong?

### DISCUSSION THREAD

**Discussion by lipis** [upptime/upptime#199] [Q&A] [answered]
https://github.com/excalidraw/upptime

From the commits we are seeing that it runs once per day: https://github.com/excalidraw/upptime/commits/master

Is that the expected behaviour? How can we make it check a bit more often?

**Comment by AnandChowdhary** [comment, marked as answer]
Please see https://github.com/upptime/upptime/discussions/112.

**Comment by wolfier74** [comment]
Please make regular interval and also add option to setup custom interval in minutes

**Comment by AnandChowdhary** [comment]
I wrote a quick blog post with some additional information about this issue: https://upptime.js.org/blog/2021/01/22/github-actions-schedule-not-working.
